### PR TITLE
Fix2

### PR DESCRIPTION
**Pull request by ViktarBaloshka**

### CODE REVIEW

File: src/main/java/com/helloworld/helloworld/service/impl/MessageServiceImpl.java [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    @PutMapping("{id}")
+    public Map<String, String> update(@PathVariable String id, @RequestBody Map<String, String> message) {
+        Map<String, String> messageFromDb = getMessage(id);
+        messageFromDb.putAll(message);
+        message.put("id", id);
+        return messageFromDb;
+    }
+
+    @DeleteMapping("{id}")
+    /*Почему когда было public void delete(@PathVariable String id) не сработало, хотя по идее
```
Comment (YanushkevichVasili) [inline src/main/java/com/helloworld/helloworld/service/impl/MessageServiceImpl.java:48]: need to return void

File: src/main/java/com/helloworld/helloworld/controller/MessageController.java [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+        return messageService.create(message);
+    }
+
+    @PutMapping("{id}")
+    public Map<String, String> update(@PathVariable String id, @RequestBody Map<String, String> message) {
+        return messageService.update(id, message);
+    }
+
+    @DeleteMapping("{id}")
+    public  Map<String, String> delete(@PathVariable String id) {
```
Comment (YanushkevichVasili) [inline src/main/java/com/helloworld/helloworld/controller/MessageController.java:37]: public ResponseEntity<Void> delete(...){
...
return ResponseEntity.noContent()..

}

File: src/main/java/com/helloworld/helloworld/service/impl/MessageServiceImpl.java (hunk 1 above)
Comment (ViktarBaloshka) [reply src/main/java/com/helloworld/helloworld/service/impl/MessageServiceImpl.java:48]: done

File: src/main/java/com/helloworld/helloworld/controller/MessageController.java (hunk 2 above)
Comment (ViktarBaloshka) [reply src/main/java/com/helloworld/helloworld/controller/MessageController.java:37]: done

**Pull request merged**